web/src/server/api/routers/generations/filterOptionsQuery.ts

```diff
@@ -109,6 +109,26 @@ export const filterOptionsQuery = protectedProjectProcedure
       `),
     ]);
 
+    const tags = await ctx.prisma.$queryRaw<
+      Array<{
+        tag: string | null;
+        count: number;
+      }>
+    >(Prisma.sql`
+        SELECT
+          tag,
+          count(*)::int AS count
```

Comment:
Can we please drop the count here to reduce latency? The counts for the filter options are quite expensive unfortunately and would be helpful to not include them for now. We did some investigation on datadog yesterday and this was one of the findings ☺️ 

Also, if you like you may include a free text option to make the filter option more flexible given we're limiting it ☺️

Reply:
Thanks for pointing out 👍  I will drop it.